docker-compose.yml

```diff
@@ -41,19 +41,31 @@ services:
             - AXIS_CTXAVAILABLETRANSPORTS=http;tchannel
 
             - BEHAVIOR_RAW=client,server,transport
+            - BEHAVIOR_SKIP_RAW=client=java+transport=tchannel,server=java+transport=tchannel
             - BEHAVIOR_JSON=client,server,transport
+            - BEHAVIOR_SKIP_JSON=client=java+transport=tchannel,server=java+transport=tchannel
             - BEHAVIOR_THRIFT=client,server,transport
+            - BEHAVIOR_SKIP_THRIFT=client=java+transport=tchannel,server=java+transport=tchannel
```

Comment:
Actually, doesn't using the prefix `BEHAVIOR_SKIP_*` mean that I can't have a behavior named `skip_foo`? What about `SKIP_*`?

Also, the example in crossdock/crossdock#83 uses `SKIP_*`. Is this PR out of date?